aap-common/proc-aap-generate-command-data-file.adoc

```diff
@@ -0,0 +1,93 @@
+[id="proc-aap-generate-command-data-file"]
+
+= Generating the data file
+
+.Procedure
+. Run the command generator.
++
+[source,bash]
+----
+$ docker run -rm -v <local_directory_data_file>:/data $IMAGE command_generator_vars <playbook_name> -output-data-file /data/<data-file>.yml
+----
++
+The outputs of this command are the command to run and the data file template.
+The data file is also saved in your `<local_data_file_directory>`.
+This is the template which you populate with your data.
++
```

Comment:
Move this plus sign inside of the ifdef::product_GCP[] for things to format correctly in the AWS doc. It's weird but it works.